themes/default/content/docs/clouds/aws/guides/vpc.md

```diff
@@ -399,10 +614,9 @@ A VPC spans all of the availability zones in your region. By default, however, t
 only use 2 of them when allocating subnets and the associated gateways. This provides fault tolerance between
```

Comment:
I was reasonably certain that AWSx uses 3 AZs (in fact, it says the first three AZs earlier in the page). We need to verify and fix whichever of these statements is incorrect.

Reply:
Great catch! I did validate that 3 seems to be the default. I will update.

![image](https://github.com/pulumi/pulumi-hugo/assets/19211559/9c3893ee-c309-4293-850a-6cbba4a52abb)

Reply:
@scottslowe updated to say 3 instead of 2

![image](https://github.com/pulumi/pulumi-hugo/assets/19211559/e911405e-c576-4b65-8f1c-933038ce33c9)

I also sanity checked in the Oregon region and it does indeed create across only 3 AZs